monitoring/accessing-third-party-monitoring-apis.adoc

```diff
@@ -9,6 +9,17 @@ toc::[]
 [role="_abstract"]
 In {product-title} {product-version}, you can access web service APIs for some third-party monitoring components from the command line interface (CLI).
 
+[IMPORTANT]
+====
+INSERT WORDING ABOUT STABILITY GUARANTEES HERE.
+
+Here's a draft:
+
+These web-service APIs are provided by third-parties. Therefore, Red Hat does not guarantee long-term compatibility with them beyond the current version.
+
+See xref:../rest_api/understanding-api-support-tiers.adoc[Understanding API tiers] and xref:../rest_api/understanding-compatibility-guidelines.adoc#api-compatibility-guidelines_compatibility-guidelines[API compatibility guidelines] for more information about API support.
+====
```

Comment:
We need to have an exhaustive list of all API services as well as their endpoints because some of them ([example](https://prometheus.io/docs/prometheus/latest/querying/api/#querying-metric-metadata)) are already marked as experimental by upstream and given the way components are configured, some endpoints aren't available.

Reply:
@simonpasquier How would we produce or obtain a list like this?

Reply:
I've created https://issues.redhat.com/browse/MON-3172